keda/values.yaml

```diff
@@ -22,7 +22,9 @@ imagePullSecrets: []
 operator:
   name: keda-operator
 
-metricsServer:  
+metricsServer:
+  # use ClusterFirstWithHostNet if `useHostNetwork: true` https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  dnsPolicy: ClusterFirst
```

Comment:
Out of curiosity, why not use "Default"? (I haven't used this before)

Reply:
`Default` is not "default" (ClusterFirst is), so setting it would change current behavior.  If you mean why not use `Default` instead of `ClusterFirstWithHostNet`, w/o the latter, you can't resolve internal services/etc (e.g. service.ns.svc.local) on a `hostNetwork: true` pod.

Reply:
That's an odd default 🙈